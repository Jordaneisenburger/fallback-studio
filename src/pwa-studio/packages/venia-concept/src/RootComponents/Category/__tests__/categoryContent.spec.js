import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import CategoryContent from '../categoryContent';

const classes = {
    root: 'a',
    title: 'b',
    gallery: 'c',
    pagination: 'd'
};

const data = {
    category: {
        description: 'test'
    },
    products: {
        items: {
            id: 1
        }
    }
};

test('renders the correct tree', () => {
    const shallowRenderer = new ShallowRenderer();
    const tree = shallowRenderer.render(
        <CategoryContent
            pageControl={{}}
            data={data}
            pageSize={6}
            classes={classes}
        />
    );

    expect(tree).toMatchSnapshot();
});
