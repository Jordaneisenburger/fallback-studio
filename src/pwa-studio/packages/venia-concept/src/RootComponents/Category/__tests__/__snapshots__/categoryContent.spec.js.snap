// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the correct tree 1`] = `
<article
  className="a"
>
  <h1
    className="b"
  >
    <div />
  </h1>
  <section
    className="c"
  >
    <Classify(Gallery)
      data={
        Object {
          "id": 1,
        }
      }
      pageSize={6}
    />
  </section>
  <div
    className="d"
  >
    <withRouter(Classify(Pagination))
      pageControl={Object {}}
    />
  </div>
</article>
`;
